docs/ecosystem.rst

```diff
@@ -41,6 +41,7 @@ Example Apps
 * `guild-operators <https://github.com/cardano-community/koios-artifacts/tree/main/files/grest>`_ - example queries and functions that the Cardano Community uses for their Guild Operators' Repository
 * `PostGUI <https://github.com/priyank-purohit/PostGUI>`_ - React Material UI admin panel
 * `prospector <https://github.com/sfcta/prospector>`_ - data warehouse and visualization platform
+* `fullstack template <https://github.com/jenstroeger/fullstack-webapp-template>`_ - a complete fullstack webapp template using PG as db and message queue, Python and Dramatiq to implement async jobs, db migrations, test runners, and more.
```

Comment:
There are some CI spellcheck errors as reported on:
https://github.com/PostgREST/postgrest/actions/runs/19247430391/job/55024751748?pr=4458

Please add those in:
https://github.com/PostgREST/postgrest/blob/bccf7ac97a8a4d821fe199cab530ec3d37c6c653/docs/postgrest.dict#L1-L196